ghcide/src/Development/IDE/LSP/Outline.hs

```diff
@@ -48,7 +48,7 @@ moduleOutline ideState DocumentSymbolParams{ _textDocument = TextDocumentIdentif
                    (defDocumentSymbol l :: DocumentSymbol)
                      { _name  = pprText m
                      , _kind  = SkFile
-                     , _range = Range (Position 0 0) (Position maxBound 0) -- _ltop is 0 0 0 0
+                     , _range = Range (Position 0 0) (Position 2147483647 0) -- _ltop is 0 0 0 0
```

Comment:
Magic constants are usually bad, a comment would be helpful, or maybe

```suggestion
                     , _range = Range (Position 0 0) (Position (2 ^31 - 1) 0) -- _ltop is 0 0 0 0
```

Reply:
I added a comment and hardcoded the constant (ghc warns and makes me add a type annotation to 31 otherwise). Also changed the usages of maxBound in test and bench which is probably not needed.